diff --git chrome/browser/extensions/BUILD.gn chrome/browser/extensions/BUILD.gn
index 2945c8b8b6bd529c0f9e49dd5c7cbba996fe8014..d5ea1a30a537a05db87e15fd5fba4f730e46bff9 100644
--- chrome/browser/extensions/BUILD.gn
+++ chrome/browser/extensions/BUILD.gn
@@ -3164,10 +3164,10 @@ source_set("browser_tests") {
 
     if (!is_chromeos) {
       # Chrome OS doesn't use background mode or download bubbles.
-      deps += [
-        "//chrome/browser/background/extensions",
-        "//chrome/browser/download/bubble",
-      ]
+      if (enable_background_mode) {
+        deps += [ "//chrome/browser/background/extensions" ]
+      }
+      deps += [ "//chrome/browser/download/bubble" ]
     }
 
     if (enable_background_mode) {
