diff --git tools/gritsettings/resource_ids.spec tools/gritsettings/resource_ids.spec
index d55db4f82a97a1161d436c5c77ffbeac0f5ec708..ca2f29fb3812585aa2083bcf91fd48f5b3448ecf 100644
--- tools/gritsettings/resource_ids.spec
+++ tools/gritsettings/resource_ids.spec
@@ -1803,6 +1803,13 @@
   # END "everything else" section.
   # Everything but chrome/, components/, content/, and ios/
 
+  "cef/libcef/resources/cef_resources.grd": {
+    "includes": [12500],
+  },
+  "cef/libcef/resources/cef_strings.grd": {
+    "messages": [12510],
+  }
+
   # Thinking about appending to the end?
   # Please read the header and find the right section above instead.
 }
