diff --git base/win/dark_mode_support.cc base/win/dark_mode_support.cc
index daecebb5070705c1eb5281a3a7df4ace6b89c119..aed51c2e6b06e3527bd6c2003e4b6a82ee4469e5 100644
--- base/win/dark_mode_support.cc
+++ base/win/dark_mode_support.cc
@@ -7,6 +7,7 @@
 #include <windows.h>
 
 #include "base/check.h"
+#include "base/command_line.h"
 #include "base/native_library.h"
 #include "base/win/win_util.h"
 #include "base/win/windows_version.h"
@@ -85,11 +86,20 @@ const DarkModeSupport& GetDarkModeSupport() {
   return dark_mode_support;
 }
 
+bool IsForcedLightMode() {
+  static bool kIsForcedLightMode =
+      base::CommandLine::ForCurrentProcess()->HasSwitch(
+          "force-light-mode");
+  return kIsForcedLightMode;
+}
+
 }  // namespace
 
 namespace base::win {
 
 bool IsDarkModeAvailable() {
+  if (IsForcedLightMode())
+    return false;
   auto& dark_mode_support = GetDarkModeSupport();
   return (dark_mode_support.allow_dark_mode_for_app ||
           dark_mode_support.set_preferred_app_mode) &&
diff --git chrome/browser/ui/views/chrome_browser_main_extra_parts_views_linux.cc chrome/browser/ui/views/chrome_browser_main_extra_parts_views_linux.cc
index 83d1f694eaf78e97c32054d4a71744d7f0f2b6b3..98de79083632c364309ae00e58ac4207a2d15589 100644
--- chrome/browser/ui/views/chrome_browser_main_extra_parts_views_linux.cc
+++ chrome/browser/ui/views/chrome_browser_main_extra_parts_views_linux.cc
@@ -66,7 +66,10 @@ void ChromeBrowserMainExtraPartsViewsLinux::ToolkitInitialized() {
 #endif
 
 #if BUILDFLAG(USE_DBUS)
-  dark_mode_manager_ = std::make_unique<ui::DarkModeManagerLinux>();
+  if (!ui::NativeTheme::IsForcedDarkMode() &&
+      !ui::NativeTheme::IsForcedLightMode()) {
+    dark_mode_manager_ = std::make_unique<ui::DarkModeManagerLinux>();
+  }
 #endif
 }
 
diff --git ui/native_theme/native_theme.cc ui/native_theme/native_theme.cc
index 049cbdd2000458534dca9b08c02264661b75f4ca..1248babf31411463c5539abf056952ae01e6acc7 100644
--- ui/native_theme/native_theme.cc
+++ ui/native_theme/native_theme.cc
@@ -297,6 +297,8 @@ void NativeTheme::NotifyOnNativeThemeUpdated() {
 
   NotifyOnNativeThemeUpdatedImpl();
 
+  color_provider_manager.AfterNativeThemeUpdated();
+
   RecordNumColorProvidersInitializedDuringOnNativeThemeUpdated(
       color_provider_manager.num_providers_initialized() -
       initial_providers_initialized);
@@ -378,6 +380,13 @@ bool NativeTheme::IsForcedDarkMode() {
   return kIsForcedDarkMode;
 }
 
+bool NativeTheme::IsForcedLightMode() {
+  static bool kIsForcedLightMode =
+      base::CommandLine::ForCurrentProcess()->HasSwitch(
+          "force-light-mode");
+  return kIsForcedLightMode;
+}
+
 bool NativeTheme::IsForcedHighContrast() {
   static bool kIsForcedHighContrast =
       base::CommandLine::ForCurrentProcess()->HasSwitch(
@@ -594,8 +603,13 @@ ColorProviderKey::ForcedColors NativeTheme::CalculateForcedColors() const {
 
 NativeTheme::PreferredColorScheme NativeTheme::CalculatePreferredColorScheme()
     const {
-  return IsForcedDarkMode() ? PreferredColorScheme::kDark
-                            : OsSettingsProvider::Get().PreferredColorScheme();
+  if (IsForcedDarkMode()) {
+    return PreferredColorScheme::kDark;
+  }
+  if (IsForcedLightMode()) {
+    return PreferredColorScheme::kLight;
+  }
+  return OsSettingsProvider::Get().PreferredColorScheme();
 }
 
 NativeTheme::PreferredContrast NativeTheme::CalculatePreferredContrast() const {
diff --git ui/native_theme/native_theme.h ui/native_theme/native_theme.h
index e131113a7d5eeea02929ff037d898f1c2f281a43..0327964801b17658f08f1ac8c4f74032bb951c85 100644
--- ui/native_theme/native_theme.h
+++ ui/native_theme/native_theme.h
@@ -539,13 +539,16 @@ class COMPONENT_EXPORT(NATIVE_THEME) NativeTheme {
   size_t system_color_version() const { return system_color_version_; }
   void IncrementSystemColorVersion() { ++system_color_version_; }
 
- protected:
-  explicit NativeTheme(SystemTheme system_theme = SystemTheme::kDefault);
-  virtual ~NativeTheme();
+  // Whether light mode is forced via command-line flag.
+  static bool IsForcedLightMode();
 
   // Whether dark mode is forced via command-line flag.
   static bool IsForcedDarkMode();
 
+ protected:
+  explicit NativeTheme(SystemTheme system_theme = SystemTheme::kDefault);
+  virtual ~NativeTheme();
+
   // Whether high contrast is forced via command-line flag.
   static bool IsForcedHighContrast();
 
