diff --git ui/accessibility/platform/BUILD.gn ui/accessibility/platform/BUILD.gn
index eadb44ecc00a1ac4f5c442eafe9ceb53279a36b1..41df3858aaa4416bcffdc74adab274c7a806d78a 100644
--- ui/accessibility/platform/BUILD.gn
+++ ui/accessibility/platform/BUILD.gn
@@ -363,6 +363,10 @@ component("platform") {
       if (use_gio) {
         configs += [ "//build/linux:gio_config" ]
       }
+
+      if (is_clang) {
+        cflags = [ "-Wno-deprecated-volatile" ]
+      }
     }
   }
 }
diff --git ui/gtk/BUILD.gn ui/gtk/BUILD.gn
index afb08ff72d342edcbdb3a00c7a8cafc21fd906a1..2c5ebcab15422f65eac7d65cd4735f6515899fb8 100644
--- ui/gtk/BUILD.gn
+++ ui/gtk/BUILD.gn
@@ -196,6 +196,10 @@ component("gtk") {
 
   # TODO: This should be removed.
   deps += [ "//ui/ozone" ]
+
+  if (is_clang) {
+    cflags = [ "-Wno-deprecated-volatile" ]
+  }
 }
 
 if (use_gtk) {
