diff --git ui/views/controls/webview/webview.cc ui/views/controls/webview/webview.cc
index 0d8af7d65e0c322191b923be3042eccecdd91a40..9fb421146a5f02dcc8e878802c2951bb9ab53b5c 100644
--- ui/views/controls/webview/webview.cc
+++ ui/views/controls/webview/webview.cc
@@ -215,6 +215,10 @@ void WebView::EnableSizingFromWebContents(const gfx::Size& min_size,
   }
 }
 
+void WebView::SetResizeBackgroundColor(SkColor resize_background_color) {
+  holder_->SetBackgroundColorWhenClipped(resize_background_color);
+}
+
 void WebView::TakeCrashedOverlayViewImpl(
     std::unique_ptr<View> crashed_overlay_view,
     ReturnCrashOverlayToOwnerCallback return_to_owner) {
diff --git ui/views/controls/webview/webview.h ui/views/controls/webview/webview.h
index 3c0f097e6084b9bbc9361ec65baf0f724e0c0f15..a5756ae40a349dd7e02d0292d296f11188bb4c9c 100644
--- ui/views/controls/webview/webview.h
+++ ui/views/controls/webview/webview.h
@@ -135,6 +135,10 @@ class WEBVIEW_EXPORT WebView : public View,
   void EnableSizingFromWebContents(const gfx::Size& min_size,
                                    const gfx::Size& max_size);
 
+  // Set the background color to use while resizing with a clip. This is white
+  // by default.
+  void SetResizeBackgroundColor(SkColor resize_background_color);
+
   // A scoped object that disconnects the webview from the accessibility tree.
   // When destroyed, it restores the previous accessibility state.
   class WEBVIEW_EXPORT ScopedAxDisconnectLock {
