diff --git ui/views/controls/textfield/textfield.cc ui/views/controls/textfield/textfield.cc
index 67061f8ed25119c92df7ad20d4284d1e1163cbe2..ead99d7842f1f51c1384a1d3ee31be4346d587ea 100644
--- ui/views/controls/textfield/textfield.cc
+++ ui/views/controls/textfield/textfield.cc
@@ -3329,6 +3329,10 @@ void Textfield::OnEnabledChanged() {
   if (GetInputMethod()) {
     GetInputMethod()->OnTextInputTypeChanged(this);
   }
+  if (GetWidget()) {
+    SetColor(GetTextColor());
+    UpdateBackgroundColor();
+  }
   UpdateDefaultBorder();
 
   if (GetWidget()) {
