diff --git chrome/browser/ui/views/frame/browser_window_property_manager_win.cc chrome/browser/ui/views/frame/browser_window_property_manager_win.cc
index 00445dbee1a207264eb601d25ed8852171fb197e..ee3dda17123579f5abfe6a1f840a8247018c046d 100644
--- chrome/browser/ui/views/frame/browser_window_property_manager_win.cc
+++ chrome/browser/ui/views/frame/browser_window_property_manager_win.cc
@@ -8,6 +8,7 @@
 #include "base/functional/bind.h"
 #include "base/strings/utf_string_conversions.h"
 #include "base/win/windows_version.h"
+#include "cef/libcef/features/features.h"
 #include "chrome/browser/browser_process.h"
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/profiles/profile_manager.h"
@@ -52,6 +53,17 @@ void BrowserWindowPropertyManager::UpdateWindowProperties() {
   const BrowserWindowInterface* browser = view_->browser();
   Profile* profile = const_cast<Profile*>(browser->GetProfile());
 
+#if BUILDFLAG(ENABLE_CEF)
+  if (browser->cef_delegate() &&
+      (browser->GetType() ==
+           BrowserWindowInterface::Type::TYPE_PICTURE_IN_PICTURE ||
+       browser->GetType() == BrowserWindowInterface::Type::TYPE_DEVTOOLS) &&
+      browser->cef_delegate()->HasViewsHostedOpener()) {
+    // Don't create a separate taskbar group.
+    return;
+  }
+#endif
+
   // Set the app user model id for this application to that of the application
   // name. See http://crbug.com/41308099.
   std::wstring app_id =
